static/js/panels.js

```diff
@@ -101,24 +104,101 @@ export function show_profile_incomplete(is_profile_incomplete) {
     }
 }
 
+function show_notification_alert() {
+    open({
+        data_process: $t({defaultMessage: "notifications"}),
+        first_step_message: $t({defaultMessage: "Zulip needs your permission to"}),
```

Comment:
You can't split up string this way as different languages have different word orders.  You need instead "Zulip needs your permission to __x__" if you're going to do this, but it'd be better to just have one string.

 https://zulip.readthedocs.io/en/latest/translating/internationalization.html

Reply:
Ohh! Thanks, the docs were very helpful!